web/src/components/atoms/ContentSection/index.tsx

```diff
@@ -0,0 +1,50 @@
+import styled from "@emotion/styled";
+import { ReactNode } from "react";
+
+export type Props = {
```

Comment:
You are missing the `className?: string` prop. Without it, this can't be used with styling libraries like Styled Components

Reply:
@KaWaite , added.
Did not know that I encountered a place where I couldn't use `styled components` but It never crossed my mind that this is the reason why.

Reply:
Yep. I think recently I fixed an issue with the Icon atom because we were trying to do styled components with it and nothing was happening because it had no className!
(once you read this, you can resolve this convo)